polaris-react/src/components/IndexTable/IndexTable.tsx

```diff
@@ -806,3 +812,4 @@ export function IndexTable({
 
 IndexTable.Cell = Cell;
 IndexTable.Row = Row;
+IndexTable.ScrollContainer = ScrollContainer;
```

Comment:
It's used https://github.com/Shopify/web/blob/main/app/sections/ShippingLabels/PurchaseShippingLabels/components/TableView/TableView.tsx#L155 but it looks like that just needs to be wholesale replaced with the IndexTable rather than the own-rolled table that's currently there. Can remove 👍